### ISSUE TITLE
Component error: auto_backup - cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)

### ISSUE THREAD

**Issue by pashdown** [jcwillox/hass-auto-backup#46]
Just started happening:
```
➜  /workspace hassio ha check
Error: Testing configuration at /config
Failed config
  General Errors: 
    - Component error: auto_backup - cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
```

**Comment by jcwillox** [comment]
I see, the issue was caused by this PR https://github.com/home-assistant/core/pull/58380, it's an easy fix though. I don't think this change has been released yet, what version of Home Assistant did you notice this on?

**Comment by pashdown** [comment]
core-2021.11.3

Seems to be gone, as of this morning, though.

**Comment by mpepping** [comment]
core-2021.11.4

```
Error: Testing configuration at /config
Failed config
  homeassistant.packages.autobackup.auto_backup:
    - Package autobackup setup failed. Component auto_backup cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
    - auto_backup: None
```

**Comment by jcwillox** [comment]
Interesting I'll release the fix for it tonight then.

**Comment by jcwillox** [comment]
Should be fixed in the latest release [0.10.1](https://github.com/jcwillox/hass-auto-backup/releases/tag/0.10.1).

**Comment by mpepping** [comment]
Works, thanks @jcwillox!
